### PR TITLE
chore: Normalize capitalization of sqldb across all files

### PR DESCRIPTION
**Pull request by SethPoulsen**

### CODE REVIEW

**Comment by SethPoulsen** [comment]
I was annoyed some files having `sqldb` and some having `sqlDb`, so I normalized it to `sqldb` because that was the majority. I'm not too bothered about which capitalization is chose in the end, I mostly just want consistency.

File: tools/replace_prairielib_paths.py [hunk 1]
```diff
@@ -40,22 +40,22 @@ def walk_directory(dirname, old_string, new_prefix, new_string=None):
 walk_directory('.', "require('@prairielearn/prairielib').error", "require('@prairielearn/prairielib/error')")
```
Comment (nwalters512) [inline tools/replace_prairielib_paths.py:40]: We should just remove this whole Python file entirely, it's not useful anymore.

File: prairielib/index.js [hunk 2]
```diff
@@ -1,4 +1,3 @@
-module.exports.sqlDb = require('./lib/sql-db');
 module.exports.sqldb = require('./lib/sql-db');
```
[the remark this answers is not in the document]
Comment (nwalters512) [reply prairielib/index.js:1]: I don't think anything is using this index file anymore; it should be safe to remove.

**Review by nwalters512 — APPROVED** [review]

File: tools/replace_prairielib_paths.py (hunk 1 above)
Comment (SethPoulsen) [reply tools/replace_prairielib_paths.py]: done

**Pull request merged**